src/hooks/useStatusesFilter.ts

```diff
@@ -0,0 +1,37 @@
+import queryString from "query-string";
+import { useLocation, useHistory } from "react-router-dom";
+
+/**
+ * @param  {string} urlParam the param that will appear in the url search, e.g. `statuses`, `baseStatuses`
+ * @return {[string[], (newValue: string[]) => void]}
+ * pass first value in return array to `value` prop of dropdown component
+ * pass second value in return array to `onChange` prop of dropdown component
+ */
+export const useStatusesFilter = (
```

Comment:
This hook read and writes array query params. I think we can change the name of this function to useArrayQueryParam since the code isn't specific to statuses or filtering